src/main/java/com/wire/helium/LoginClient.java

```diff
@@ -231,38 +257,6 @@ public void removeCookies(String token, String password) throws HttpException {
 
     }
 
-    public NotificationList retrieveNotifications(String client, UUID since, String token, int size) throws HttpException {
```

Comment:
Where is this gone?

Reply:
This was moved to `src/main/java/com/wire/helium/API.java:557` as it makes more sense to have it on a general API file than specifically on the login client